apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.html.ts

```diff
@@ -0,0 +1,167 @@
+import { z } from 'zod';
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { AssessmentQuestionSchema } from '../../../lib/db-types';
+import { idsEqual } from '../../../lib/id';
+
+export const ManualGradingQuestionSchema = AssessmentQuestionSchema.extend({
+  qid: z.string(),
+  title: z.string(),
+  number: z.string().nullable(),
+  alternative_group_number: z.number(),
+  alternative_group_size: z.coerce.number(),
+  num_instance_questions: z.coerce.number(),
+  num_instance_questions_to_grade: z.coerce.number(),
+  num_instance_questions_assigned: z.coerce.number(),
+  num_instance_questions_unassigned: z.coerce.number(),
+  assigned_graders: z
+    .array(z.object({ user_id: z.number(), name: z.string().nullable(), uid: z.string() }))
+    .nullable(),
+  actual_graders: z
+    .array(z.object({ user_id: z.number(), name: z.string().nullable(), uid: z.string() }))
+    .nullable(),
+  num_open_instances: z.coerce.number(),
+});
+export type ManualGradingQuestion = z.infer<typeof ManualGradingQuestionSchema>;
+
+export function ManualGradingAssessment({
+  resLocals,
+  questions,
+  num_open_instances,
+}: {
+  resLocals: Record<string, any>;
+  questions: ManualGradingQuestion[];
+  num_open_instances: number;
+}) {
+  return html`<!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../partials/head') %>", resLocals)}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../../partials/navbar'); %>", resLocals)}
+        <main id="content" class="container-fluid">
+          ${renderEjs(
+            __filename,
+            "<%- include('../../partials/assessmentSyncErrorsAndWarnings'); %>",
+            resLocals,
+          )}
+          ${renderEjs(__filename, "<%- include('../../partials/assessmentOpenInstancesAlert') %>", {
+            ...resLocals,
+            num_open_instances,
+          })}
+          <div class="card mb-4">
+            <div class="card-header bg-primary text-white">
+              ${resLocals.assessment_set.name} ${resLocals.assessment.number}: Manual Grading Queue
+            </div>
+
+            <div class="table-responsive">
+              <table id="instanceQuestionGradingTable" class="table table-sm table-hover">
+                <thead>
+                  <tr>
+                    <th>Question</th>
+                    <th>QID</th>
+                    <th>Auto Points</th>
+                    <th>Manual Points</th>
+                    <th colspan="2">Submissions to grade</th>
+                    <th>Grading assigned to</th>
+                    <th>Graded by</th>
+                    <th>Actions</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  ${questions.map(
+                    (question) =>
+                      html` <tr>
```

Comment:
Typically we start HTML on a new line, unless it can fit in a single line. Unfortunately, Prettier won't make this change automatically.

```suggestion
                      html`
                        <tr>
```

Reply:
Actually I tried doing that and prettier undid it...

Reply:
Yeah, Prettier is a little tricky. You'll need to add a newline before the closing backtick too.

Reply:
Done in 62d97cd7f.